controller/kubernetes_pod_controller.go

```diff
@@ -329,6 +322,20 @@ func (kc *KubernetesPodController) handleWorkloadPodDeletionIfCSIPluginPodIsDown
 	return nil
 }
 
+func (kc *KubernetesPodController) isStorageNetworkForRWXVolume() (bool, error) {
```

Comment:
NIT: This is an exact replica of a function in the share-manager controller. Can we make it a datastore method instead? Or a utility function that takes a datastore?

Reply:
Done.